docs/products/flink/howto/create-flink-applications.rst

```diff
@@ -1,61 +1,13 @@
-   * In the **Table SQL** section, enter the statement that will be used to create the table. 
-   * Optionally, select **Run** to view how data is being pulled from the data source. This could take some time based on the data and the connection. 
-   * Select **Add table**. 
-7. Select **Next** to add the sink table, and then select **Add your first sink table**. 
-   
-   .. note::   
-    Since this is your first application, there are currently no other applications where you can import sink tables.
-    
-8.  In the **Add new sink table** screen, 
-    
-    * Select **Integrated service** from the drop-down list. 
-    * In the **Table SQL** section, enter the statement that will be used to create the table.  
-    * Select **Add table**. 
-9.  Select **Next** to enter the **SQL statement** that transforms the data from the source stream. Optionally, select **Run** to view how data is being pulled from the data source. 
-10. Select **Save and deploy later** to save the application. You will see the application on the landing page that provides you with an overview of your application. 
-
-    .. image:: /images/products/flink/application_landingpage_view.png
-        :scale: 50 %
-        :alt: Application landing page with an view of source table, SQL statement and sink table
-    
-11. Select **Create deployment**. On the **Create new deployment** screen, 
-    
-    * Choose the version you wish to deploy. The default version for the first deployment is **Version: 1**. 
-    * Choose the :doc:`savepoint </docs/products/flink/concepts/savepoints>` from where you want to deploy. No savepoints are available for the first application deployment. 
-    * Use the toggle for **Restart on failure** to enable or disable the option of automatically restarting a Flink job in case it fails. 
-    * Enter the number of `parallel instances <https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/execution/parallel/>`_ you want to have for the task. 
-12. Select **Deploy without a savepoint** to deploy your application. 
-13. The deployment status will show **Initializing: version 1** and then **Running: version 1**.
-
-Your first application is now created and deployed, and you can view the data related to the actions the application needs to perform in your sink source.
+:doc:`Aiven for Flink applications </docs/products/flink/concepts/flink-applications>` in Aiven for Apache Flink® servers as a container that includes everything connected to a Flink job, including source and sink connections and data processing logic. 
+
+With the `Aiven Console <https://console.aiven.io/>`_, you can create applications that run SQL queries or deploy custom JARs, catering to diverse data processing requirements. The console's guided wizard simplifies the application configuration process, from selecting source and sink tables for SQL applications to uploading and managing JAR files for custom job execution.
+
+
```

Comment:
Can we have a banner here emphasising JARs are in limited availability and to contact your sales team if you are interested in this functionality